lib/web/mage/adminhtml/browser.js

```diff
@@ -52,22 +52,25 @@ define([
                 content = '<div class="popup-window" id="' + windowId + '"></div>',
                 self = this;
 
-            if (this.modalLoaded === true &&
-                options &&
-                self.targetElementId &&
-                self.targetElementId === options.targetElementId
+            if (options
+                && self.targetElementId
+                && self.targetElementId === options.targetElementId
             ) {
                 if (typeof options.closed !== 'undefined') {
                     this.modal.modal('option', 'closed', options.closed);
                 }
                 this.modal.modal('openModal');
 
                 return;
+            } else if (typeof options === 'undefined' 
```

Comment:
Check to undefined it isn't the best solution. You can check just like `if (options)` or in case if you really need to check to undefined ( maybe in some cases it's really needed ) we should use _.isUndefined method from underscore library because `typeof` has some side effects

Reply:
ok